### PR TITLE
Remove broken AppNexus examples.

### LINKED ISSUE #3589: Appnexus Ads are broken
App Nexus ads in amp seems to be broken since awhile with the following error:

```
Uncaught TypeError: Cannot read property 'anq' of undefined
```

image

@mkendall07 any idea if you can take a look and see if you can fix this? You can reproduce this by running `gulp` and navigating to `ads.amp.max.html` example.

### PR DESCRIPTION
**Pull request by lannka**
They were removed in #3638 to fix #3589, but brought back by #4202 
Remove again.

### CODE REVIEW

File: examples/ads.amp.html [hunk 1]
```diff
@@ -195,20 +195,7 @@ <h2 id="amoad" class="broken">AMoAd</h2>
       data-sid="62056d310111552c951d19c06bfa71e16e615e39493eceff667912488bc576a6">
   </amp-ad>
 
-  <h2 id="appnexus" class="broken">AppNexus single ad with tagid (placement id)</h2>
```
Comment (mkhatib) [inline examples/ads.amp.html]: Can you track down who contributed the appnexus ad and file a bug against them to add a valid example so we make sure we don't break their ads.

File: examples/ads.amp.html [hunk 2]
```diff
@@ -195,20 +195,7 @@ <h2 id="amoad" class="broken">AMoAd</h2>
...
-  </amp-ad>
-
-  <h2 class="broken">AppNexus single ad with member and code</h2>
-  <amp-ad width=300 height=250
-      type="appnexus"
-      data-member="958"
-      data-code="inv_code_test">
-  </amp-ad>
-
-  <h2>AppNexus with JSON based configuration multi ad</h2>
```
Comment (mkhatib) [inline examples/ads.amp.html]: Keep this? This is referring to the next ad, no?

**Comment by mkhatib** [comment]
LGTM please file a bug and mention people who contributed appnexus to ask them to add a proper example.

File: examples/ads.amp.html (hunk 1 above)
Comment (lannka) [inline examples/ads.amp.html]: I believe it was removed by the contributor himself in #3638 .
I accidentally brought them back in #4202

File: examples/ads.amp.html (hunk 2 above)
Comment (lannka) [inline examples/ads.amp.html]: Ha, it's not removed. See line 198 on the right side.

**Pull request merged**